centreon/www/front_src/src/Resources/Listing/useLoadResources/index.ts

```diff
@@ -277,6 +277,10 @@ const useLoadResources = (): LoadResources => {
   };
 
   useEffect(() => {
+    const user = useAtomValue(userAtom);
+    if(!user.is_admin){
+      return ;
+    }
```

Comment:
So if you are not an admin, you cannot load resources status, right?

Reply:
yes that's it

Reply:
@Exodiaaaa  Consider using navigationAtom to verify the existence of a page with label : 'Resources Status'. you can use some functions from https://ramdajs.com/docs/ to achieve this

Reply:
I think you would consider manage this in the react router